system-test/system_test.sh

```diff
@@ -12,13 +12,28 @@ retry() {
 
 cd $(dirname $0)
 
+if [[ -z "$BINARY_HOST" ]]; then
+  BUILD_BINARIES="true"
+fi
+
 for i in 6 8 10 11; do
   # Test Linux support for the given node version.
-  retry docker build -f Dockerfile.linux --build-arg NODE_VERSION=$i -t node$i-linux .
-  docker run -v $PWD/..:/src node$i-linux /src/system-test/test.sh
+  retry docker build \
```

Comment:
The indentation of the wrapped lines is inconsistent here and below - somewhere zero, somewhere 1, somewhere 2. It should be four spaces, actually I think.

Reply:
Done.